services/src/main/scala/org/apache/gearpump/cluster/main/Services.scala

```diff
@@ -37,18 +38,58 @@ object Services extends App with ArgumentsParser {
   }
 
   override val options: Array[(String, CLIOption[Any])] = Array(
-    "master" -> CLIOption[String]("<host1:port1,host2:port2,host3:port3>", required = true))
+    "master" -> CLIOption[String]("<host1:port1,host2:port2,host3:port3>", required = true),
```

Comment:
I don't see necessity to add new cli option.

You can use java -D property, it will be parsed by typesafe Config and override the setting in gear.conf.